<ui version="4.0" >
 <class>KCupsOptionsJobWidget</class>
 <widget class="QWidget" name="KCupsOptionsJobWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Job</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="jobControlGroupBox" >
     <property name="title" >
      <string>Job Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="jobHoldLabel" >
        <property name="text" >
         <string>Scheduled printing:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="KComboBox" name="jobHoldComboBox" />
      </item>
      <item row="0" column="3" >
       <widget class="QTimeEdit" name="jobHoldTimeEdit" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="jobBillingLabel" >
        <property name="text" >
         <string>Billing information:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3" >
       <widget class="KLineEdit" name="jobBillingLineEdit" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="jobPriorityLabel" >
        <property name="text" >
         <string>Job priority:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="jobPrioritySpinBox" >
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>180</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1" >
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>35</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="jobOptionsGroupBox" >
     <property name="title" >
      <string>Job Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QTableWidget" name="jobOptionsTable" >
        <property name="rowCount" >
         <number>10</number>
        </property>
        <property name="columnCount" >
         <number>2</number>
        </property>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text" >
          <string>Option</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Value</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0" >
       <spacer name="verticalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer name="verticalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
